docs/install/quick-start-guide.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Quick Start Guide - Non-Production"
+linkTitle: "Quick Start Guide"
+weight: 1
+---
+
+This guide will provide step by step instructions on how to install Ondat onto your cluster, with our helm chart, for a non-production environment.
+
+> ⚠️ This guide is for a non-production installation. Please follow our [other installation guides](https://docs.ondat.io/docs/install/) for a production-ready installation of Ondat
+
+## Prerequisites
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [helm](https://helm.sh/docs/intro/install/)
+
+This guide assumes you already have a Kubernetes cluster, with **at least** 3 worker nodes.
+
+### This guide has been tested on the following Kubernetes distributions
```

Comment:
Customer faced so I would suggest "This steps work on the following..." instead of "has been tested"